src/components/confirm/confirm.test.tsx

```diff
@@ -0,0 +1,217 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import Confirm from "../confirm";
+
+jest.mock("../../__internal__/utils/helpers/guid", () => () => "guid-12345");
+
+describe("Confirm", () => {
```

Comment:
suggestion: remove unnecessary describe block

Reply:
Should also add that if tests aren't within a describe block then they should be `test()` as opposed to `it()` based on Kent C Dodd's recommendations

See: https://kentcdodds.com/blog/avoid-nesting-when-youre-testing#inline-it